### PR TITLE
Add entry for 5.4.0-m1

### PR DESCRIPTION
**Pull request by jburel**
# What this PR does

Add internal version history
The text will be reviewed/improved when the publicly release 5.4.0-m1

cc @pwalczysko @mtbc

### CODE REVIEW

File: history.txt [hunk 1]
```diff
@@ -1,6 +1,19 @@
...
+5.4.0-m1 (July 2017)
+--------------------
+
+Developer preview release - **only intended as a developer preview for
+updating code before the full public release of 5.4.0. Use at your own risk**.
+
+Changes include:
+
+- added support for new security privileges to allow users to perform some
+  admin tasks  without having admin privileges e.g. import for others
```
Comment (mtbc) [inline history.txt]: double space

File: history.txt [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+- added specific installation instruction for Debian 9
```
Comment (mtbc) [inline history.txt]: instructions

**Comment by pwalczysko** [comment]
Reads fine.

**Comment by jburel** [comment]
Thanks @pwalczysko 
Merging

**Pull request merged**